cylc/flow/exceptions.py

```diff
@@ -81,7 +78,9 @@ class InputError(CylcError):
 
 class CylcConfigError(CylcError):
     """Generic exception to handle an error in a Cylc configuration file."""
-    # TODO: reference the configuration element causing the problem
+    # TODO: reference the configuration el
+    
+    ement causing the problem
```

Comment:
???

Reply:
VDI had a fit whilst I was trying to use the online de-conflict tool.